book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] Please add a method to create a many-to-many relationship between the person and the book. It should look something like this:
```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end

```